src/hkml_view_mails.py

```diff
@@ -334,9 +335,11 @@ def do_export(data, answer, selection):
     try:
         answer = int(answer)
     except:
-        print('wrong input.  Return to hkml')
         time.sleep(1)
         hkml_view.shell_mode_end(slist)
+        err = 'wrong input. Return to hkml'
+        print(err)
```

Comment:
After `shell_mode_end()` is invoked, we are already in curses mode, so users may not see this `print()`.  I think we should do the print() before the `shell_mode_end()`.  Not a blocker of this PR, though.